web_chatter_right/static/src/js/chatter_right.js

```diff
@@ -0,0 +1,30 @@
+odoo.define('web_chatter_right.Chatter', function (require) {
```

Comment:
2 options: 

a. call it `web_chatter_side`. Arabic readers may want it left. 

b. Do not use a new addon. Instead, add the features to `web_responsive`. (I prefer this one)